lib/rails_stdout_logging/rails.rb

```diff
@@ -1,5 +1,5 @@
 module RailsStdoutLogging
-  class StdoutLogger < ::Logger
+  class StdoutLogger < ::ActiveSupport::Logger
```

Comment:
I think that should be fine, let's go ahead with that change.

I don't know how 3.x implements logger silencing. Do you know @carlosantoniodasilva ?